### PR TITLE
Add script to count backend method usage

### PR DESCRIPTION
**Pull request by Jikstra**
```
# messageList.getMessage [17]
- ./components/dialogs/MessageDetail.tsx [2]
- ./components/dialogs/FullscreenMedia.tsx [1]
- ./components/dialogs/ChatAuditLogDialog.tsx [2]
- ./components/map/MapComponent.tsx [1]
- ./components/helpers/ChatMethods.ts [1]
- ./stores/chat.ts [10]
# contacts.getContact [9]
- ./components/dialogs/CreateChat.tsx [1]
- ./components/chat/ChatList.tsx [1]
- ./components/contact/ContactList.tsx [1]
- ./components/helpers/OpenQrUrl.tsx [2]
- ./components/helpers/ChatMethods.ts [1]
- ./components/map/MapComponent.tsx [1]
- ./stores/settings.ts [2]
# contacts.createChatByContactId [7]
- ./components/dialogs/ViewProfile.tsx [1]
- ./components/Sidebar.tsx [1]
- ./components/helpers/ChatMethods.ts [1]
- ./components/helpers/OpenQrUrl.tsx [1]
- ./components/message/messageFunctions.ts [1]
- ./components/message/MessageMarkdown.tsx [2]
# chatList.getFullChatById [7]
- ./components/dialogs/MailtoDialog.tsx [1]
- ./components/dialogs/ViewGroup.tsx [1]
- ./components/chat/ChatListContextMenu.tsx [2]
- ./components/helpers/ChatMethods.ts [1]
- ./components/message/MessageMarkdown.tsx [1]
- ./stores/chat.ts [1]
# setProfilePicture [7]
- ./components/dialogs/Settings-Profile.tsx [7]
# messageList.getMessageIds [6]
- ./components/dialogs/ChatAuditLogDialog.tsx [1]
- ./stores/chat.ts [5]
# contacts.createContact [6]
- ./components/dialogs/CreateChat.tsx [2]
- ./components/chat/ChatList.tsx [1]
- ./components/helpers/OpenQrUrl.tsx [1]
- ./components/message/MessageMarkdown.tsx [1]
- ./experimental.ts [1]
# messageList.getDraft [5]
- ./components/composer/Composer.tsx [2]
- ./components/dialogs/MailtoDialog.tsx [1]
- ./components/message/MessageMarkdown.tsx [1]
- ./components/message/messageFunctions.ts [1]
# messageList.getMessages [5]
- ./components/dialogs/ChatAuditLogDialog.tsx [1]
- ./stores/chat.ts [4]
# settings.setDesktopSetting [4]
- ./components/screens/AccountListScreen.tsx [1]
- ./components/message/MessageListAndComposer.tsx [2]
- ./stores/settings.ts [1]
# messageList.setDraft [4]
- ./components/composer/Composer.tsx [1]
- ./components/dialogs/MailtoDialog.tsx [1]
- ./components/message/messageFunctions.ts [1]
- ./components/message/MessageMarkdown.tsx [1]
# getContacts2 [4]
- ./components/contact/ContactList.tsx [4]
# context.getConnectivity [3]
- ./components/dialogs/Settings-Profile.tsx [1]
- ./components/dialogs/Settings-Connectivity.tsx [1]
- ./components/ConnectivityToast.tsx [1]
# chat.setMuteDuration [3]
- ./components/dialogs/MuteChat.tsx [1]
- ./components/helpers/ChatMethods.ts [1]
- ./stores/chat.ts [1]
# chat.getQrCode [3]
- ./components/dialogs/CreateChat.tsx [1]
- ./components/dialogs/ViewGroup.tsx [1]
- ./components/Sidebar.tsx [1]
# backup.export [3]
- ./components/dialogs/Settings-Backup.tsx [3]
# settings.saveBackgroundImage [2]
- ./components/dialogs/Settings-Appearance.tsx [2]
# settings.getDesktopSettings [2]
- ./components/screens/AccountListScreen.tsx [1]
- ./stores/settings.ts [1]
# settings.setConfig [2]
- ./stores/settings.ts [2]
# messageList.removeDraft [2]
- ./components/composer/Composer.tsx [2]
# messageList.getMessagesFromIndex [2]
- ./stores/chat.ts [2]
# login.addAccount [2]
- ./ScreenController.tsx [2]
# chat.getNextMedia [2]
- ./components/dialogs/FullscreenMedia.tsx [2]
# chat.setVisibility [2]
- ./components/helpers/ChatMethods.ts [2]
# chat.delete [2]
- ./components/composer/Composer.tsx [1]
- ./components/helpers/ChatMethods.ts [1]
# chat.modifyGroup [2]
- ./components/dialogs/MessageListProfile.tsx [1]
- ./components/dialogs/ViewGroup.tsx [1]
# chat.getQrCodeSVG [2]
- ./components/dialogs/ViewGroup.tsx [1]
- ./components/Sidebar.tsx [1]
# contacts.lookupContactIdByAddr [2]
- ./components/helpers/OpenQrUrl.tsx [1]
- ./components/message/MessageMarkdown.tsx [1]
# chatList.selectChat [2]
- ./stores/chat.ts [2]
# joinSecurejoin [2]
- ./components/helpers/OpenQrUrl.tsx [2]
# webxdc.getWebxdcDiskUsage [1]
- ./components/dialogs/Settings-Webxdc.tsx [1]
# webxdc.clearWebxdcDOMStorage [1]
- ./components/dialogs/Settings-Webxdc.tsx [1]
# extras.writeClipboardToTempFile [1]
- ./components/composer/Composer.tsx [1]
# extras.setTheme [1]
- ./components/dialogs/Settings-Appearance.tsx [1]
# extras.getAvailableThemes [1]
- ./components/dialogs/Settings-Appearance.tsx [1]
# extras.getActiveTheme [1]
- ./ThemeManager.tsx [1]
# extras.setLocale [1]
- ./App.tsx [1]
# extras.getLocaleData [1]
- ./App.tsx [1]
# context.getConnectivityHTML [1]
- ./components/dialogs/Settings-Connectivity.tsx [1]
# context.maybeNetwork [1]
- ./components/ConnectivityToast.tsx [1]
# stickers.getStickers [1]
- ./components/composer/EmojiAndStickerPicker.tsx [1]
# settings.estimateAutodeleteCount [1]
- ./components/dialogs/Settings-Autodelete.tsx [1]
# settings.keysExport [1]
- ./components/dialogs/Settings-ManageKeys.tsx [1]
# settings.keysImport [1]
- ./components/dialogs/Settings-ManageKeys.tsx [1]
# messageList.saveMessageHTML2Disk [1]
- ./components/message/messageFunctions.ts [1]
# messageList.msgIds2SearchResultItems [1]
- ./components/chat/ChatList.tsx [1]
# messageList.searchMessages [1]
- ./components/chat/ChatListHelpers.tsx [1]
# messageList.getFirstUnreadMessage [1]
- ./stores/chat.ts [1]
# messageList.forwardMessage [1]
- ./components/dialogs/ForwardMessage.tsx [1]
# messageList.downloadFullMessage [1]
- ./components/message/messageFunctions.ts [1]
# messageList.getMessageInfo [1]
- ./components/dialogs/MessageDetail.tsx [1]
# messageList.deleteMessage [1]
- ./stores/chat.ts [1]
# messageList.sendSticker [1]
- ./components/composer/EmojiAndStickerPicker.tsx [1]
# messageList.sendMessage [1]
- ./stores/chat.ts [1]
# login.getLastLoggedInAccount [1]
- ./ScreenController.tsx [1]
# login.getAllAccountIds [1]
- ./ScreenController.tsx [1]
# login.getAccountSize [1]
- ./components/screens/AccountListScreen.tsx [1]
# login.selectAccount [1]
- ./ScreenController.tsx [1]
# locations.getLocations [1]
- ./stores/locations.ts [1]
# chat.accept [1]
- ./components/composer/Composer.tsx [1]
# chat.sendVideoChatInvitation [1]
- ./components/helpers/ChatMethods.ts [1]
# chat.setChatEphemeralTimer [1]
- ./components/dialogs/DisappearingMessages.tsx [1]
# chat.getChatEphemeralTimer [1]
- ./components/dialogs/DisappearingMessages.tsx [1]
# chat.markNoticedChat [1]
- ./stores/chat.ts [1]
# chat.createGroupChat [1]
- ./components/dialogs/CreateChat.tsx [1]
# chat.setProfileImage [1]
- ./components/dialogs/CreateChat.tsx [1]
# chat.addContactToChat [1]
- ./components/dialogs/CreateChat.tsx [1]
# chat.setName [1]
- ./components/dialogs/CreateChat.tsx [1]
# chat.leaveGroup [1]
- ./components/helpers/ChatMethods.ts [1]
# chat.getEncryptionInfo [1]
- ./components/dialogs/EncryptionInfo.tsx [1]
# chat.getChatMedia [1]
- ./components/Gallery.tsx [1]
# contacts.deleteContact [1]
- ./components/dialogs/CreateChat.tsx [1]
# contacts.getEncryptionInfo [1]
- ./components/dialogs/EncryptionInfo.tsx [1]
# contacts.getContacts [1]
- ./components/chat/ChatList.tsx [1]
# contacts.getContactIds [1]
- ./components/contact/ContactList.tsx [1]
# contacts.changeNickname [1]
- ./components/dialogs/ViewProfile.tsx [1]
# contacts.getBlocked [1]
- ./components/dialogs/UnblockContacts.tsx [1]
# contacts.blockContact [1]
- ./components/helpers/ChatMethods.ts [1]
# contacts.unblockContact [1]
- ./components/dialogs/UnblockContacts.tsx [1]
# chatList.getChatListEntryMessageIdForChatId [1]
- ./components/chat/ChatList.tsx [1]
# backup.import [1]
- ./components/screens/WelcomeScreen.tsx [1]
# autocrypt.continueKeyTransfer [1]
- ./components/dialogs/EnterAutocryptSetupMessage.tsx [1]
# autocrypt.initiateKeyTransfer [1]
- ./components/dialogs/Settings-Encryption.tsx [1]
# checkQrCode [1]
- ./components/helpers/OpenQrUrl.tsx [1]
# extras.setThemeFilePath [0]
# settings.serverFlags [0]
# messageList.messageIdToJson [0]
# locations.setLocation [0]
# chat.block [0]
# chat.getChatContacts [0]
# chatList.getGeneralFreshMessageCounter [0]
# chatList.onChatModified [0]
# chatList.getSelectedChatId [0]
# getProfilePicture [0]
```

### CODE REVIEW

File: bin/find_delta_remote.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ *
+ * @param {string[]} args arguments for the command
+ * @param {import('child_process').SpawnOptionsWithoutStdio} [options]
+ */
+async function run(command, args, options, listener = undefined) {
+  return new Promise((resolve, reject) => {
+    console.log(`- Executing "${command} ${args.join(' ')}"`)
+    p.stdout.pipe(process.stdout)
+  })
+}
```
Comment (Simon-Laux) [inline bin/find_delta_remote.js:16]: whats this used for?

**Review by Simon-Laux — APPROVED** [review]
LGTM

File: bin/find_delta_remote.js (hunk 1 above)
Comment (obsoleszenz) [reply bin/find_delta_remote.js:16]: That's dead code, i first copied the run function from another of our scripts

Comment (Jikstra) [reply bin/find_delta_remote.js:16]: Oh that's dead code, will remove it

**Review by Simon-Laux — APPROVED** [review]

**Pull request merged**